public/js/components/SourceTabs.js

```diff
@@ -12,6 +13,7 @@ const actions = require("../actions");
 const { isEnabled } = require("../../../config/feature");
 
 require("./SourceTabs.css");
+require("./Dropdown.css");
```

Comment:
Agreed (thought I was just meaning to extract the dropdown already in this component), but in that case I think we should just have the dropdown css in the source tabs css. I don't think we should have arbitrary CSS files; they exist as an implementation for a component, so it's weird to have standalone ones.